### DISCUSSION TITLE
Dev new backend for sigmac

### DISCUSSION THREAD

**Discussion by nNipsx-Sec** [SigmaHQ/sigma#3026] [Ideas]
Hi Team,

I want to dev new backend for sigmac, have anyone document guideline for this.

Thanks

**Comment by Neo23x0** [comment]
Don't do that for Sigmac. Sigmac will be deprecated somewhen in the near future. 

All future development should be based on pySigma. 

https://github.com/SigmaHQ/pySigma

Example backend

https://github.com/SigmaHQ/pySigma-backend-splunk
